### PR TITLE
Alihan/taskString1

### PR DESCRIPTION
**Pull request by AleBaydo**
please don't pull the request to the original branch, just check it out.

### CODE REVIEW

**Comment by Asylbash** [comment]
Sei stato bravissimo!

File: src/test/java/alihan/string1_test/MakeTagsTest.java [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    public void testMakeTags (String tag, String word, String expected){
+        MakeTags makeTags = new MakeTags();
+        String result = makeTags.makeTags(tag,word);
+        Assert.assertEquals(result,expected);
+
+    }
+    @DataProvider
+    public Object[][] makeTags(){
+        return new Object[][]{
+
```
Comment (Asylbash) [inline src/test/java/alihan/string1_test/MakeTagsTest.java:20]: tut body poprav

**Review by Asylbash — APPROVED** [review]

File: src/test/java/alihan/string1_test/MakeTagsTest.java (hunk 1 above)
Comment (AleBaydo) [reply src/test/java/alihan/string1_test/MakeTagsTest.java:20]: @Asylbash here is just a page break on Github )))

Comment (Asylbash) [reply src/test/java/alihan/string1_test/MakeTagsTest.java:20]: nado je vse poprobyvat

**Pull request closed without merging**